modular_skyrat/modules/liquids/code/mop.dm

```diff
@@ -0,0 +1,25 @@
+/**
+ * The procedure for remove water from turf
+ *
+ * The object is called from liquid_interaction element.
+ * The procedure check range of mop owner and tile, then check reagents in mop, if reagents volume < mop capacity - liquids absorbs from tile
+ * In another way, input a chat about mop capacity
+ * Arguments:
+ * * the_mop - Mop what used to absorb liquids(???)
```

Comment:
I'm still not entirely sure why that needs to be an argument, would it be possible to just not have it anymore? I'm not entirely familiar with elements.

Reply:
it's need to element, I tried to remove it, but the mop did not suck water anymore.